plugin/caddyfile/testdata/labels/quotes.txt

```diff
@@ -0,0 +1,12 @@
+caddy                = localhost
+caddy.0_with_spaces    = "a b c d"
+caddy.1_without_spaces = "abcd"
+caddy.2_multiple       = "a b c d" "abcd"
+caddy.3_back           = `a b c d`
+----------
+localhost {
+	with_spaces "a b c d"
+	without_spaces abcd
+	multiple "a b c d" abcd
+	back "a b c d"
```

Comment:
This isn't right I don't think. It should preserve the backticks. They're usually used intentionally to wrap JSON text like:

```
caddy.3_back = `{"some":"json"}`
```

Reply:
I updated that test.
During parsing, it parses into a single token. I don't think we need to preserve the original quote used to define that token.

But when serializing that token back, we need to make sure it generates a valid Caddyfile, and today it does:
https://github.com/lucaslorentz/caddy-docker-proxy/blob/master/plugin/caddyfile/marshal.go#L44

Reply:
If the token has whitespace, or quotes, it is printed back to caddyfile inside double quotes and any quotes are escaped.

Reply:
:+1: